<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Satyam" failures="1" tests="1" name="test1" time="37.353" errors="0" timestamp="2020-11-03T18:25:06 IST">
  <testcase classname="Test.APIUITest" name="ValidateUIandAPIResult" time="23.573">
    <failure type="java.lang.RuntimeException" message="Temperature is not with in variance range. UI value: 304.6111 | API value :297.21">
      <![CDATA[java.lang.RuntimeException: Temperature is not with in variance range. UI value: 304.6111 | API value :297.21
at Test.APIUITest.ValidateUIandAPIResult(APIUITest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- ValidateUIandAPIResult -->
</testsuite> <!-- test1 -->
